src/Acuminator/Acuminator.Analyzers/StaticAnalysis/GraphDeclarationTypeParameter/GraphDeclarationTypeParameterAnalyzer.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Acuminator.Utilities.Roslyn.Semantic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace Acuminator.Analyzers.StaticAnalysis.GraphDeclarationTypeParameter
```

Comment:
@metallmonster, I think it's better to name it `PXGraphDeclarationTypeParameter` and rename all other related classes/namespaces
It's inconvenient when all other graph analyzer names start with "PXGraph" and this one doesn't

Reply:
@SENya1990 ok